source/Hardpoint.cpp

```diff
@@ -33,7 +33,7 @@ using namespace std;
 
 namespace {
 	// Create all the effects in the given list, at the given location, velocity, and angle.
-	void CreateEffects(const map<const Effect *, int> &m, Point pos, Point vel, Angle angle, vector<Visual> &visuals)
+	void CreateEffects(const map<const Effect *, int> &m, Point pos, Point vel, Angle angle, list<Visual> &visuals)
```

Comment:
It doesn’t let me comment further above, but didn’t you forget to add an `#include <list>`?

Reply:
It's already included in `Hardpoint.h`, which is included by this file.

Reply:
~~In theory it should probably be included here too~~